uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -379,12 +380,13 @@ define([
         }
 
         if (missingRequiredFields.length) {
-          var notification = CRM.alert(missingRequiredFields.join(', '),
-            missingRequiredFields.length === 1 ? 'Required field' : 'Required fields', 'error');
+          var notificationTitle = missingRequiredFields.length === 1 ? 'Required field' : 'Required fields';
+          var missingFields = missingRequiredFields.join(', ');
+          var alert = notification.alert(notificationTitle, missingFields);
 
-          notification && $timeout(function () {
-            notification.close();
-            notification = null;
+          alert && $timeout(function () {
+            alert.close();
```

Comment:
Thanks for catching! I think should pass the {expires: 5} options instead to `notification`.